skopt/space.py

```diff
@@ -486,3 +486,39 @@ def transformed_bounds(self):
                 b.extend(dim.transformed_bounds)
 
         return b
+
+
+class DictSpace(Space):
+
+    def __init__(self, dimensions):
+        self.keys = sorted(list(dimensions.keys()))
+        super(DictSpace, self).__init__([dimensions[k] for k in self.keys])
+
+    def rvs(self, n_samples=1, random_state=None):
+        return [
+            dict(zip(self.keys, sample))
+            for sample in super(DictSpace, self).rvs(n_samples, random_state)
+        ]
+
+    def transform(self, X):
+        return super(DictSpace, self).transform([
+            [sample[k] for k in self.keys] for sample in X
+        ])
+
+    def inverse_transform(self, Xt):
+        return [
+            dict(zip(self.keys, sample))
+            for sample in super(DictSpace, self).inverse_transform(Xt)
+        ]
+
+
+def make_space(dimensions):
+    if isinstance(dimensions, Space):
+        return dimensions
```

Comment:
with this change it is now possible to pass to *_minimize a `Space` instance directly, which was not possible before, let me know if you don't like that for some reason and I can remove this branch from the function